src/components/todo/ToDoPriority.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="relative w-30 phone:hidden">
+    <div
+      class="flex cursor-pointer mb-0 px-8 py-1 justify-center my-3 rounded-full font-semibold text-white"
+      :class="priorityColorChange(todo.priority)"
+      @click="toggleDropdown"
+    >
+      {{ localTodo.priority }}
+    </div>
+    <ul
+      class="absolute bg-white mt-2 mr-2 rounded-xl font-medium border border-black border-2 right-0"
+      :class="getDropdownState"
+    >
+      <div class="flex flex-col items-start">
+        <li>
+          <p
+            class="block px-3 text-black cursor-pointer"
+            @click="updatePriority('Low', index)"
```

Comment:
Please create const variable for the priority and set the  `Low`, 'Medium' and `High` there. In this case use can use the variable to set the value of the priority instead of writing down the same string more times.